src/test/java/com/redhat/cloud/policies/app/OapiTest.java

```diff
@@ -34,12 +33,14 @@
 import java.nio.file.StandardCopyOption;
 import java.util.Map;
 
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Star import

Reply:
Replaced all star imports with the last commit.